sentry-android-core/src/main/java/io/sentry/android/core/SentryAndroid.java

```diff
@@ -124,6 +120,24 @@ public static synchronized void init(
 
             configuration.configure(options);
 
+            // if SentryPerformanceProvider was disabled or removed, we set the App Start when
+            // the SDK is called.
+            // pre-starfish: fill-back the app start time to the SDK init time
+            if (options.isEnableStarfish()) {
+              final @NotNull TimeSpan appStartTimeSpan =
+                  AppStartMetrics.getInstance().getAppStartTimeSpan();
+              if (appStartTimeSpan.hasNotStarted()
```

Comment:
please use the BuildInfoProvider for testability instead of `android.os.Build.VERSION.SDK_INT`